CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
If I understand your suggestion correctly, I believe that is how the current implementation works.
The `rightBarButtonItem` adopts the tintColor for the overall color scheme unless a `progressRingTintColor` is applied.